### PR TITLE
fix deploy

### PR DESCRIPTION
**Pull request by katSchmid**

### CODE REVIEW

File: index.cjs [hunk 1]
```diff
@@ -4,7 +4,12 @@
 var path = require("path");
 
 // If in 'production' mode, use the combined/minified/optimized version of Cesium
-if (process.env.NODE_ENV === "production") {
+if (
+  process.env.NODE_ENV === "production" ||
+  process.env.NODE_ENV === "deploy-staging" ||
+  process.env.NODE_ENV === "deploy-production" ||
+  process.env.NODE_ENV === "deploy-dev"
```
Comment (chris-cooper) [inline index.cjs]: I think just `dev`, `staging` and `production` (i.e. drop the `deploy-`): https://github.com/PropellerAero/lambda-surface-math/blob/dev/codeship-services.yml#L19

**Review by chris-cooper — COMMENTED** [review]
Don't forget to bump the version number

File: index.cjs (hunk 1 above)
Comment (katSchmid) [reply index.cjs]: ok

**Review by chris-cooper — APPROVED** [review]
Looks good :+1:

**Pull request merged**